src/main/kotlin/com/vanniktech/maven/publish/Utils.kt

```diff
@@ -0,0 +1,12 @@
+package com.vanniktech.maven.publish
+
+import org.gradle.api.Project
+import org.gradle.api.publish.PublishingExtension
+import org.gradle.plugins.signing.SigningExtension
+import java.util.concurrent.Callable
+
+internal val Project.signing get() = extensions.getByType(SigningExtension::class.java)
```

Comment:
Make all of them inline?

Reply:
done fd48f06ffc1ea74009fc24d500f7828982a44ac9